src/containers/notification/components/desktop/NotificationBlock.tsx

```diff
@@ -231,7 +232,7 @@ const NotificationBlock = (props: NotificationBlockProps) => {
         <div className={styles.body}>{body}</div>
         {!header && (
           <div className={styles.menuContainer}>
-            <Menu menu={props.menuProps}>
+            <Menu menu={props.menuProps} zIndex={zIndex.NAVIGATOR_POPUP + 1}>
```

Comment:
Thoughts on making this another zIndex field instead of adding to the popup so that we know don't do this somewhere else and break this?

Reply:
Yeah I was going back and forth on this, I think having a separate key for the `NAVIGATOR_POPUP_OVERFLOW_POPUP` is better. Added!